config/routes.rb

```diff
@@ -3,10 +3,12 @@
 Rails.application.routes.draw do
   get 'home', to: 'pages#home', as: :home
 
-  get 'sign_up', to: 'registration#new'
-  post 'sign_up', to: 'registration#create'
+  resources :users
```

Comment:
not sure that all actions needs for now, check it and use `resources :users, only: %i[create destroy]`